### ISSUE TITLE
getting started isn't working

### ISSUE THREAD

**Issue by janat08** [QQuick/Transcrypt#472]
Open a command prompt in the demos/hello directory and type transcrypt hello.py. Then click on hello.html to load it into your browser.

I suspect demos/hello is suppose to be in transcrypt install folder, which I don't want to know where it is?

This will equally disappoint: http://www.transcrypt.org/ without any indication about if the code should generate which it really should along with necessary files for your demo.

**Comment by JdeH** [comment]
Valid remark, this requires improvement.

**Comment by akaihola** [comment]
For `hello.html`, the instructions at http://www.transcrypt.org/home show:
>`<script src="__javascript__/hello.js"></script>`

For my own test project, this wouldn't work, and on my Firefox Quantum 60.0 when loading `hello.html` from a local folder, I needed to use this instead:
```html
<script type="module" src="__target__/main.js"></script>
```

**Comment by g40** [comment]
Having similar problems here with Chrome 69.0.3497.92 on Windows, with Python 3.7 x64 and Transcrypt 3.7.4. 

The pages are being served via the Python HTTP server. When `hello.html` is modified to

```<script type="module" src="__target__/hello.js"></script>``` 

Chrome complains with:

```Failed to load module script: The server responded with a non-JavaScript MIME type of "text/plain". Strict MIME type checking is enforced for module scripts per HTML spec.```

Rather sad to see this not working. There is much promise here. Any thoughts welcomed.

**Comment by akaihola** [comment]
My error message on Chrome 69.0.3497.100 as well as Brave 0.24.0 on Linux is different:

>Access to Script at 'file:///tmp/hello/\_\_target\_\_/hello.js' from origin 'null' has been blocked by CORS policy: Invalid response. Origin 'null' is therefore not allowed access.

The work-around is to serve the files with HTTP instead of loading them locally:

- ```console
  cd hello
  python -m http.server
  ```
- open http://localhost:8000/hello.html in the browser

**Comment by JdeH** [comment]
The instructions on the home page are still very unclear, will change that right away!
Indeed you HAVE to run a http server.
This is a new security feature of Chrome to prevent Cross Origin Resource Sharing.

**Comment by JdeH** [comment]
The instructions should be correct and more clear now...
Right?

**Comment by metaperl** [comment]
On Cygwin, the instructions to for this sample work fine. However on Windows 10, I get the same error as @g40 - "Failed to load module script: The server responded with a non-JavaScript MIME type of "text/plain". Strict MIME type checking is enforced for module scripts per HTML spec."

**Comment by ghtmpn** [comment]
**Some Windows boxes have wrong content type set in registry for `.js`**

Solution/info:
https://github.com/labstack/echo/issues/1038#issuecomment-410294904

![capture](https://user-images.githubusercontent.com/30050961/52317931-412fca00-29d3-11e9-9b4f-942aa16f0c01.PNG)

This seems to be a common issue with python+mimetypes+windows, so I would consider adding this as a special note for windows users, since debugging it down to mimetypes.py is not what you usually do for getting started.

**Comment by ashleyo** [comment]
Still a problem with Firefox (65), works fine with Edge, haven't tried Chrome. 
You get a 'Loading failed for the module with source “http://localhost:8000/__target__/hello.js”' reported in the console.
It does indeed appear to be an issue with mimetypes, Firefox is doing it correctly (so far as I can see from the spec) and insisting on the correct mimetype and the other browsers are saying .... wtf, let's do it anyway ....
The *blame* though is attached to the python server which seems to be getting it wrong as noted above by metaperl.

If you have node installed you can:
```
>npm install -g http-server
>http-server
```
or with npx installed just
```
>npx http-server
```
to run a different server which solves the problem

**Comment by ghtmpn** [comment]
I don’t think that python server is getting it wrong, because it seems that registry lookup is a standard way for determining mime types on msw. My fresh installation of w10 at work showed that .js was application/javascript. Probably some installer/app changes that later, but it is hard to find which and why.
